test/runnable/opover2.d

```diff
@@ -1363,7 +1364,7 @@ void test9453()
 
 struct S9496
 {
-	static S9496* ptr;
+        static S9496* ptr;
```

Comment:
It was a tab in the original. My git script converts tabs to spaces, and the tab got converted to 8. Fixed.